functions/Get-DbaAgBackupHistory.ps1

```diff
@@ -141,7 +144,9 @@ function Get-DbaAgBackupHistory {
         [switch]$IncludeMirror,
         [ValidateSet("Full", "Log", "Differential", "File", "Differential File", "Partial Full", "Partial Differential")]
         [string[]]$Type,
-        [switch]$EnableException
+        [switch]$EnableException,
+        [ValidateSet("FirstLsn",  "DatabaseBackupLsn", "LastLsn")]
```

Comment:
Because EnableException should always be the last parameter, please move the new one up one element.

Reply:
Thanks for the feedback, updated.